lib/vty.c

```diff
@@ -4102,16 +4076,14 @@ int vty_mgmt_send_get_req(struct vty *vty, bool is_config,
 }
 
 int vty_mgmt_send_get_data_req(struct vty *vty, uint8_t datastore,
-			       LYD_FORMAT result_type, uint8_t flags,
-			       uint8_t defaults, const char *xpath)
+			       uint8_t format, uint8_t flags, uint8_t defaults,
+			       const char *xpath)
 {
-	LYD_FORMAT intern_format = result_type;
-
 	vty->mgmt_req_id++;
 
 	if (mgmt_fe_send_get_data_req(mgmt_fe_client, vty->mgmt_session_id,
-				      vty->mgmt_req_id, datastore,
-				      intern_format, flags, defaults, xpath)) {
```

Comment:
please leave the `intern_format` name as it is highlighting that it can be different (LYD_LYB) from the users requested format.